test_libs/pyspec/eth2spec/test/sanity/test_blocks.py

```diff
@@ -91,7 +92,7 @@ def test_empty_epoch_transition(state):
 #     block = build_empty_block_for_next_slot(state)
 #     block.slot += spec.SLOTS_PER_EPOCH * 5
 #     sign_block(state, block, proposer_index=0)
-#     yield 'blocks', [block], [spec.BeaconBlock]
+#     yield 'blocks', [block], List[spec.BeaconBlock]
```

Comment:
Any reason for commenting out this test case?

Tips: you can skip a test case with:
```python
@pytest.mark.skip(reason="<THE REASON>")
```

Reply:
Comment is from a previous PR. The 5 epoch long transition is horribly slow. Polishing these few tests is good for a future PR, maybe it's fast enough when we have some basic SSZ caching.